migration/main.py

```diff
@@ -99,62 +125,56 @@ async def main(api: API, account_id: int, term_ids: list[int],
                 logger.info(f'Source tool: {source_tool}')
                 logger.info(f'Target tool: {target_tool}')
 
-                progress = TrioProgress(total=len(courses))
+                progress = TrioProgress(total=len(courses), unit='courses')
                 trio.lowlevel.add_instrument(progress)
                 async with trio.open_nursery() as nursery:
                     for course in courses:
-                        nursery.start_soon(
-                            migrate_tool_for_course,
-                            api,
-                            course,
-                            source_tool,
-                            target_tool
-                        )
+                        nursery.start_soon(migrate_tool_for_course, api,
+                                           course, source_tool, target_tool)
                 trio.lowlevel.remove_instrument(progress)
 
 
 def run():
-    logging.basicConfig(
-        level=logging.INFO,
-        style='{',
-        format='{asctime} | {levelname} | {module}:{lineno} | {message}')
-
     logger.info('Starting migration…')
 
     # get configuration (either env. variables, cli flags, or direct input)
     root_dir: str = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
     env_file_name: str = os.path.join(root_dir, 'env')
 
     if os.path.exists(env_file_name):
-        logger.info(f'Setting environment from file "{env_file_name}".')
+        logger.info(f'Setting environment from file {repr(env_file_name)}.')
         load_dotenv(env_file_name, verbose=True)
     else:
-        logger.info(f'File "{env_file_name}" not found.  '
-                     'Using existing environment.')
+        logger.info(f'File {repr(env_file_name)} not found.  '
+                    'Using existing environment.')
 
     logger.info('Parameters from environment…')
 
     # Set up logging
     log_level_default = logging.INFO
```

Comment:
Looks like this code was getting the log level from the env, does this still work? (my earlier comment)

Reply:
Yes.  That line sets the default log levels in case the params don't specify them.  After fetching the log level params or getting the default value, several lines a little further down set the values in the loggers.